### PR TITLE
Fix GitHub Actions

### PR DESCRIPTION
**Pull request by carlosggarcia**
Solve #110.

At the moment, I have updated TJPCov to use NaMaster v2. There is a remaining error with the binning implementation that was implemented recently.

### CODE REVIEW

**Comment by carlosggarcia** [comment]
Ok. This now is working but I had to comment out the cache and specify the coverage output. The latter means that coveralls see a 94% coverage instead of 99%. Not sure what the difference is but I wouldn't spend more time investigating it, at the moment. We can open a issue if needed.

**Review by damonge — APPROVED** [review]
Minor comment. Will approve anyway, since I assume fixing that is trivial and doesn't require another review (and I need to disappear now!)

**Comment by carlosggarcia** [comment]
Thanks. Addressed. I will merge when the tests pass

**Pull request merged**
